src/NLopt.jl

```diff
@@ -243,7 +243,7 @@ end
 ############################################################################
 # converting error results into exceptions
 
-type ForcedStop <: Exception end
+mutable struct ForcedStop <: Exception end
```

Comment:
`struct` is fine

Reply:
(This code pre-dates the existence of immutable types, which is why it used `type`.)

Reply:
ok